module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -902,7 +902,8 @@ public function getMyHolds($patron)
                     $entry['pickup_library_id'] ?? null
                 ),
                 'create' => $this->convertDate($entry['hold_date'] ?? null),
-                'expire' => $available ? null : $expirationDate,
+            '__create' => $entry['hold_date'] ?? null,
```

Comment:
Minor nitpick: the indentation here seems inconsistent. Is it possible you are using tabs instead of spaces somewhere, or something like that?